editor/scene/scene_tree_editor.cpp

```diff
@@ -2415,6 +2538,16 @@ HashMap<Node *, SceneTreeEditor::CachedNode>::Iterator SceneTreeEditor::NodeCach
 	return I;
 }
 
+HashMap<Node *, SceneTreeEditor::CachedNode>::Iterator
+SceneTreeEditor::NodeCache::find_by_item(TreeItem *p_item) {
+	for (auto I = cache.begin(); I; ++I) {
```

Comment:
`auto` is [not allowed](https://contributing.godotengine.org/en/latest/engine/guidelines/cpp_usage_guidelines.html#auto-keyword)